meow/frontend/src/components/EditPost/EditForm.js

```diff
@@ -119,8 +119,26 @@ class EditForm extends React.Component {
     setFieldsValue({ tags });
   };
 
+  handleClickpub = e => {
+    this.setState({
+      ...this.state,
+      click_pubready: e.target.checked
+    });
+  };
+
   render() {
     const { getFieldDecorator, getFieldsValue } = this.props.form;
+
+    let flag_urlwarn = null;
+    if (
+      getFieldsValue(["pub_ready_online"]).pub_ready_online &&
+      (this.props.story_url == null || this.props.story_url.trim() == "")
+    ) {
+      flag_urlwarn = <p className="no-url-warning">Warning: no url was entered</p>;
```

Comment:
The message seems good! I would just add one small phrase that lets them know can still publish the meow even tho there is no url.